src/lib/intent/editor-layout/editor-layout-delete-field-group.ts

```diff
@@ -0,0 +1,45 @@
+import Intent from '../base-intent'
+import { PlanMessage } from '../../interfaces/plan-message'
+import chalk from 'chalk'
+import { EditorLayoutDeleteFieldGroupAction } from '../../action/editor-layout/editor-layout-delete-field-group'
+
+export default class EditorLayoutDeleteFieldGroupIntent extends Intent {
+  isEditorInterfaceIntent () {
+    return true
+  }
+  isEditorLayoutUpdate () {
+    return true
+  }
+  isGroupable () {
+    return true
+  }
+  groupsWith (other: Intent): boolean {
+    return other.isGroupable()
+      && other.isEditorLayoutUpdate()
+      && this.isSameContentType(other)
+  }
+  endsGroup (): boolean {
+    return false
+  }
+  shouldSave (): boolean {
+    return false
+  }
+  shouldPublish (): boolean {
+    return false
+  }
+  toActions () {
+    return [
```

Comment:
Why doesn't this save the editor interface?

Reply:
I’ll leave it as is but in the next PR I’ll add a validation to disallow creating editor layout / group controls if they exist.

Reply:
Added